config/config.py

```diff
@@ -158,14 +169,14 @@ def build_config():
         "tauES_1prong0pizeroUp",
         {"global": {"tau_ES_shift_DM0": 1.002}},
         [[TauPtCorrection, "global"]],
-        [[LVMu1, "mt"]],
+        [[LVMu1, "mt"], [VetoMuons, "mt"]],
```

Comment:
is this the sanitizing, such that the ES shift is not propagated to the vetoMuons ?

Maybe we should also add the argument names here, like we do for producer instances, to make it easier to understand why this is set here.

Reply:
Maybe just for the sanetize_producers argument, which only pops up in some cases? I think repeating all arguments would be too much (e.g. looking at the JEC-shifts-config)

Reply:
yep, fine with me